Vagrantfile

```diff
@@ -419,7 +421,11 @@ Vagrant.configure("2") do |config|
           s.path = "scripts/linux/cleanup-k8s-and-cni.sh"
         end
         host.vm.provision "mount-shared", type: "shell", run: "never" do |s|
-          s.inline = "umount /vagrant; mount -t vboxsf vagrant /vagrant/"
+            if(vagrant_provider == 'virtualbox')
+                s.inline = "umount /vagrant; mount -t vboxsf vagrant /vagrant/"
+            elsif(vagrant_provider == 'libvirt')
+                s.inline = "mount -t nfs -o 'vers=3' 192.168.121.1:" + vagrant_root + " /vagrant"
```

Comment:
Shouldn't this IP be dynamic?

Reply:
I've associated the IP to the management_network_address used by libvirt with a variable definition in the Vagrantfile

Reply:
now the definition of the management_network_address used by libvirt is in default.yaml
the ip address of the nfs server is cabled in Vagrant file line 40
libvirt_management_host_address = '192.168.121.1'
to be fixed by @ferrarimarco

Reply:
now fixed by calculating the libvirt_management_host_address as the first address of the subnet